x-pack/test/functional/apps/snapshot_restore/home_page.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { KibanaFunctionalTestDefaultProviders } from '../../../types/providers';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getPageObjects, getService }: KibanaFunctionalTestDefaultProviders) => {
+  const pageObjects = getPageObjects(['common', 'snapshotRestore']);
+  const log = getService('log');
+
+  describe('Home page', function() {
+    this.tags('smoke');
+    before(async () => {
+      await pageObjects.common.navigateToApp('snapshotRestore');
+    });
+
+    it('Loads the app', async () => {
+      const appTitle = 'Snapshot Repositories';
```

Comment:
https://github.com/elastic/kibana/pull/39193 was merged, so when this PR is updated with latest master, this should be changed to `Snapshot and Restore`